plugins/com.google.cloud.tools.eclipse.sdk/src/com/google/cloud/tools/eclipse/sdk/messages.properties

```diff
@@ -0,0 +1,3 @@
+InstallJobName=Installing the Cloud SDK...
+InstallStarting=Installing the Cloud SDK... (may take up to several minutes)
+sdkModificationLocked=The Cloud SDK is being used or modified; cannot install now.
```

Comment:
being used or modified --> in use. or how about
Cannot upgrade because the Cloud SDK is in use